app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,7 @@ $govuk-new-link-styles: true;
 @import 'govuk_publishing_components/components/image-card';
 @import 'govuk_publishing_components/components/input';
 @import 'govuk_publishing_components/components/inset-text';
+@import 'govuk_publishing_components/components/intervention';
```

Comment:
The intervention component also has a JavaScript file, I'd recommend including that as well in this commit.

Reply:
Done